doc/content/gcmr/GCMR_Multiphysics_models.md

```diff
@@ -0,0 +1,148 @@
+    input = fmg
+    subdomains = '10 100 102 103 200 201 300 400 401 500 600 8000 8001'
+    extra_element_id_names = 'material_id equivalence_id'
+    extra_element_ids = '803 802 810 806 807 807 804 801 801 808 809 805 805;
+                         803 802 810 806 807 807 804 801 801 808 809 805 805'
+  []
+  [coarse_mesh]
+    type = GeneratedMeshGenerator
+    dim= 3
+    nx = 10
+    ny = 10
+    nz = 20
+    xmin = -0.1
+    xmax =  0.1
+    ymin = -0.1
+    ymax =  0.1
+    zmin =  0.
+    zmax =  2.
+  []
+  [assign_coarse_id]
+    type = CoarseMeshExtraElementIDGenerator
+    input = fmg_id
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
+
+!media media/gcmr/Fig_2.jpg
+      style=display: block;margin-left:auto;margin-right:auto;width:60%;
+      id=Fig_2
+      caption= Three-D mesh representation of the GC-MR assembly
+
+For analysis of the multi-physics, BISON and SAM were coupled with Griffin. A different mesh file were used `BISON_mesh.e` in which the coolant channels where removed from the 3-D solid mesh, as the coolant channels were handled separately by SAM. It should be noticed that BISON and SAM are coupled through coolant channel surface.   
+
+
+## Griffin Model 
+
+
+For modeling the GC-MR Assembly in 3-D, the neutron transport equation was solved by Griffin using the discrete ordinates SN method for the angular discretization while the discontinuous finite method (DFEM) method was used for the spatial discretization [!citep](Ahmed_ANS_2022) [!citep](Nicolas_ANS_2022). The coarse mesh finite element method (CMFD) acceleration was used to speed up the DFEM solver. On-the-fly coarse mesh generation is done with Griffin. The 3-D GC-MR assembly was solved with reflective boundary condition radially and vacuum boundary conditions axially on the top and bottom reflector boundaries. The homogenized multi-group cross-sections were generated using Serpent-2 with ENDF/B-VII.1 data library and then converted to XML-format cross section file and they functionalized with the fuel temperature for eleven energy groups structure. These multi-group structures were generated at the beginning of the cycle. Griffin reads the XML-format cross-section file, `XS_Griffin`, and a mesh file `Griffin_mesh.e` that specifies the 3-D mesh which has the same geometry as the one used in the cross-section generation code. In both the steady-state and the transient Griffin calculations DFEM-SN with CMFD was used. Because good accuracy was obtained with the eleven energy groups structure, only small numbers of polar and azimuthal angles were used with the SN solver: 
+
+!listing  
+[SN]
+    scheme = DFEM-SN
+    family = MONOMIAL
+    order = FIRST
+    AQtype = Gauss-Chebyshev
+    NPolar = 2
+    NAzmthl = 3
+    NA = 2
+    n_delay_groups = 6
+    sweep_type = asynchronous_parallel_sweeper
+    using_array_variable = true
+    collapse_scattering  = true
+    hide_angular_flux = true
+[]
+
+
```

Comment:
I think you need to include a separate section about the cross section preparation and the Griffin commands on how they are processed. Also how do you `has the same geometry as the one used in the cross-section generation code` is very important to the modelists and needs to be explained further.

Reply:
I've added more explanation.